.github/workflows/test.yml

```diff
@@ -308,6 +308,7 @@ jobs:
     runs-on: ubuntu-20.04
     container: timberio/ci_image
     needs: changes
+    if: ${{ needs.changes.outputs.source == 'true' }}
```

Comment:
Some of the below steps actually have guards to ensure they only run on relevant changes. Would you want to just add a guard to `make check-scripts` and `make check-version` below? It does seem like we want these checks to run outside of just `source` changes (like `helm`).

Reply:
Yeah, on second thought I think this isn't really much of a win. I'll close this.